src/hazelweb/gui/Cell.re

```diff
@@ -56,6 +56,22 @@ let focus = () => {
   JSUtil.force_get_elem_by_id("cell")##focus;
 };
 
+let caret_from_pos = (x: float, y: float): Vdom.Node.t => {
+  let pos_attr =
+    Vdom.Attr.create(
+      "style",
+      "left: "
+      ++ string_of_float(x)
+      ++ "0px; top: "
+      ++ string_of_float(y)
+      ++ "0px",
```

Comment:
Could you see if you can get this working using [`Css_gen`](https://ocaml.janestreet.com/ocaml-core/latest/doc/virtual_dom/Css_gen/index.html)?